api/db/seeds/data/team-certification/data-builder.js

```diff
@@ -338,33 +342,69 @@ async function _createV3Session({ databaseBuilder }) {
 }
 
 async function _createPublishedSession({ databaseBuilder }) {
+  const sessionDate = new Date();
   await tooling.session.createPublishedSession({
     databaseBuilder,
     sessionId: PUBLISHED_SESSION_ID,
     accessCode: 'SCOS78',
     address: '1 rue Certification pro',
     certificationCenter: 'Centre de certification pro',
     certificationCenterId: PRO_CERTIFICATION_CENTER_ID,
-    date: new Date(),
-    description: 'Pro published session',
+    date: sessionDate,
+    description: 'une description',
```

Comment:
Je trouve ca bien de pas avoir une description generique
```suggestion
    description: 'Pro published session'
```